fission-web-server/library/Fission/Web/Server/Types.hs

```diff
@@ -305,76 +303,6 @@ instance MonadDNSLink Server where
 instance MonadLinkedIPFS Server where
   getLinkedPeers = asks ipfsRemotePeers
 
-instance MonadIPFSPinner Server where
```

Comment:
Less useful when we're making all requests against the entire cluster.

There's a version where we only request against one node and have a separate cluster manager.

...ugh, okay I just convinced myself (again) that's a good idea. Will put this back. Sorry, Daniel 😛

Reply:
Made an `MonadIPFSCluster` class instead